admin_guide/install/deploy_router.adoc

```diff
@@ -190,15 +190,35 @@ $ oadm router --default-cert=cloudapps.router.pem \
 ----
 ====
 
+[NOTE]
+====
+Browsers only consider wildcards valid for subdomains one
+level deep. So in this example, the certificate would be valid for
+_a.cloudapps.example.com_ but not for _a.b.cloudapps.example.com_.
+====
+
+
+[[special-notes]]
+*Special Note About Secured Routes*
+
+Currently, password protected key files are not supported. HAProxy prompts
+for a password upon starting and does not have a way to automate this process.
+To remove a passphrase from a keyfile, you can run:
+
+----
+# openssl rsa -in <passwordProtectedKey.key> -out <new.key>
+----
+
 == Deploying a Customized HAProxy Router
 
-The HAProxy router is based on a *golang* template. This template generates the
-HAProxy configuration file. If you want to customize a router to meet your
-needs, you can change the template file, build a new docker image, and run a
-customized router.
+The HAProxy router is based on a
+link:http://golang.org/pkg/text/template/[*golang* template] that
+generates the HAProxy configuration file from a list of routes. If you
+want a customized template router to meet your needs, you can customize
+the template file, build a new docker image, and run a customized router.
 
 One common case for this might be implementing new features within the
-application back ends. For example, it might be desirable in a highly-available
+application backends. For example, it might be desirable in a highly-available
```

Comment:
Style guide thing to have "back end" / "front end" as two words when a noun. :neckbeard: